src/list/manage/components/list-sort.js

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Localized } from "fluent-react";
+import PropTypes from "prop-types";
+import React from "react";
+
+import classNames from "../../../common";
+
+import styles from "./list-sort.css";
+
+export default class ListSort extends React.Component {
+  // TODO: do we need to do something with className?
```

Comment:
Probably not, unless it's expected that someone using this will want to select a style variant. Could be YAGNI for now.